cric-predict/src/Components/Login.js

```diff
@@ -2,34 +2,44 @@ import React from 'react';
 import '../Stylesheets/App.css';
 import { Row, Col, Form, Button } from 'react-bootstrap';
 import axios from 'axios';
+import {Redirect} from "react-router-dom";
 
 
 class LoginComponent extends React.Component {
 
     constructor(props) {
         super(props);
-        this.state = {username: '', password: ''};
+        this.state = {username: '', password: '', loggedin: false};
```

Comment:
what would happen if the user is logged in and lands on the login page? he/she would need to log in again?

Reply:
Well, I guess I will have to add some checks for the access token before displaying login page